src/data/faq_de.json

```diff
@@ -1290,13 +1290,13 @@
                                     "<b>Ablauf</b>",
-                                    "<ul><li>Sie erreichen die Hotline ausschließlich telefonisch.\nRufnummer: 0800-7540002 (in Deutschland) bzw. +49.30.49875402 (international)\nWann Sie die Hotline erreichen, prüfen Sie bitte hier:\n<a href='#international_phone_numbers' target='_blank'>FAQ: So erreichen Sie uns</a></li><li>Schritt 1: Beantworten Sie die Fragen des Hotline-Mitarbeiters.</li><li>Schritt 2: Sie erhalten einen Rückruf von diesem Mitarbeiter, um Ihnen die TAN mitzuteilen. Diese TAN ist eine Stunde lang gültig.</li><li>Schritt 3: Entfernen Sie, falls registriert, den Test ohne Ergebnis aus der Corona-Warn-App.</li><li>Schritt 4: Im Home Screen der CWA wählen Sie die Schaltfläche „Weiter“ in der Funktion „Sie lassen sich testen?“\nUnter der Frage „Ihr PCR-Test war positiv?“ finden Sie die Funktion „TAN für PCR-Test eingeben“, wählen Sie diese Funktion und geben Sie die 10-stellige TAN ein, die Ihnen mitgeteilt wurde.</li><li>Ihr positives Testergebnis ist nun in der App registriert. Nun können Sie über die App andere warnen.</li></ul>",
+                                    "<ul><li>Sie erreichen die Hotline ausschließlich telefonisch.\nRufnummer: 0800 7540002 (in Deutschland) bzw. +49 30 498 75402 (international).\nWann Sie die Hotline erreichen, prüfen Sie bitte hier:\n<a href='#international_phone_numbers' target='_blank'>FAQ: So erreichen Sie uns</a>.</li><li>Schritt 1: Beantworten Sie die Fragen des Hotline-Mitarbeiters.</li><li>Schritt 2: Sie erhalten einen Rückruf von diesem Mitarbeiter, um Ihnen die TAN mitzuteilen. Diese TAN ist eine Stunde lang gültig.</li><li>Schritt 3: Entfernen Sie, falls registriert, den Test ohne Ergebnis aus der Corona-Warn-App.</li><li>Schritt 4: Unter Status der CWA wählen Sie die Schaltfläche „Weiter“ in der Funktion „Sie lassen sich testen?“.\nUnter der Frage „Ihr PCR-Test war positiv?“ finden Sie die Funktion „TAN für PCR-Test eingeben“, wählen Sie diese Funktion und geben Sie die 10-stellige TAN ein, die Ihnen mitgeteilt wurde.</li><li>Ihr positives Testergebnis ist nun in der App registriert. Nun können Sie über die App andere warnen.</li></ul>",

```

Comment:
Put Status in ''

```suggestion
                                    "<ul><li>Sie erreichen die Hotline ausschließlich telefonisch.\nRufnummer: 0800 7540002 (in Deutschland) bzw. +49 30 498 75402 (international).\nWann Sie die Hotline erreichen, prüfen Sie bitte hier:\n<a href='#international_phone_numbers' target='_blank'>FAQ: So erreichen Sie uns</a>.</li><li>Schritt 1: Beantworten Sie die Fragen des Hotline-Mitarbeiters.</li><li>Schritt 2: Sie erhalten einen Rückruf von diesem Mitarbeiter, um Ihnen die TAN mitzuteilen. Diese TAN ist eine Stunde lang gültig.</li><li>Schritt 3: Entfernen Sie, falls registriert, den Test ohne Ergebnis aus der Corona-Warn-App.</li><li>Schritt 4: Unter 'Status' der CWA wählen Sie die Schaltfläche „Weiter“ in der Funktion „Sie lassen sich testen?“.\nUnter der Frage „Ihr PCR-Test war positiv?“ finden Sie die Funktion „TAN für PCR-Test eingeben“, wählen Sie diese Funktion und geben Sie die 10-stellige TAN ein, die Ihnen mitgeteilt wurde.</li><li>Ihr positives Testergebnis ist nun in der App registriert. Nun können Sie über die App andere warnen.</li></ul>",
```

Reply:
Same reason here for not putting "Status" in quotes.